src/pages/dynamic/index.tsx

```
@@ -1,14 +1,16 @@
-"use client";
+export function getServerSideProps() {
+  return "component";
+}
 
-export default function DynamicImport({ props }: { props: any }) {
+export default function DynamicImport({ props }: { props: {} }) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)